### ISSUE TITLE
cook server not starting

### ISSUE THREAD

**Issue by Haelle** [cooklang/CookCLI#39]
I'm having an issue with the `cook server` ; here is the error (Ubuntu 20.04) : 

```sh
cook: dl-call-libc-early-init.c:37: _dl_call_libc_early_init: Assertion `sym != NULL' failed.
[1]    15474 abort (core dumped)  cook server
```

It seems to be a compatibility problem old/mordern Linux distribution maybe more [here](https://stackoverflow.com/questions/66098387/how-to-run-an-old-binary-on-modern-gnu-linux-distribution).

-> but it only happen on v0.1.3 ; v0.1.2 works fine

**Comment by dubadub** [comment]
@Haelle thanks for reporting an issue. Yes, I think you're right. Most likely it's related to this change https://github.com/cooklang/CookCLI/pull/37.

**Comment by dubadub** [comment]
As a temporary workaround, building from source should resolve the issue.

**Comment by dubadub** [comment]
@Haelle this issue should be fixed in `v0.1.4`, please feel free to re-open the issue if not. Thanks!
